src/components/Selectors/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, useState } from 'react';
+import { CheckBoxProps } from '../';
+import { mergeClasses } from '../../../shared/utilities';
+
+import styles from './checkbox.module.scss';
+
+export const CheckBox: FC<CheckBoxProps> = ({
+    ariaLabel,
+    checked = false,
+    defaultChecked,
+    disabled = false,
+    name,
+    value = '',
+    id = `checkbox-${value}-${Math.random().toString(36).slice(2)}`,
+    onChange,
+}) => {
+    const [isChecked, setIsChecked] = useState<boolean>(checked);
+
+    const checkBoxCheckClassNames: string = mergeClasses([
+        styles.checkmark,
+        { [styles.disabled]: disabled },
+    ]);
+
+    const toggleChecked = (): void => {
+        if (!disabled) setIsChecked(!isChecked);
+    };
+
+    const handleKeyDown = (event: React.KeyboardEvent) => {
```

Comment:
You'll need to do something similar for RadioButton. Currently, when I tab to a RadioButton and press the arrow keys nothing happens until I click on the page to `blur()` the focused one. You may need to handle that focus scenario for RadioButton so people don't get stuck.